<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Manager</class>
 <widget class="QMainWindow" name="UI_Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spielplan Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="ManagerMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>Spielplan</string>
    </property>
    <addaction name="actionLoadSpielplan"/>
    <addaction name="actionSaveSpielplan"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMannschaften">
    <property name="title">
     <string>Mannschaften</string>
    </property>
    <addaction name="actionLoadMannschaftsliste"/>
    <addaction name="actionSaveMannschaftsliste"/>
    <addaction name="separator"/>
    <addaction name="actionEditMannschaftsliste"/>
   </widget>
   <addaction name="MenuFile"/>
   <addaction name="menuMannschaften"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadSpielplan">
   <property name="text">
    <string>Spielplan laden</string>
   </property>
  </action>
  <action name="actionSaveSpielplan">
   <property name="text">
    <string>Spielplan speichern</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionLoadMannschaftsliste">
   <property name="text">
    <string>Mannschaftsliste laden</string>
   </property>
  </action>
  <action name="actionSaveMannschaftsliste">
   <property name="text">
    <string>Mannschaftsliste speichern</string>
   </property>
  </action>
  <action name="actionEditMannschaftsliste">
   <property name="text">
    <string>Mannschaftsliste editieren</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
